src/test/java/utils/ApiUtils.java

```diff
@@ -0,0 +1,44 @@
+package utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Charsets;
+import com.google.gson.Gson;
+import feign.Response;
+import models.Post;
+import models.User;
+import org.apache.commons.io.IOUtils;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+
+public class ApiUtils {
+
+    public static String returnJsonString(Response response) throws IOException {
```

Comment:
the method should not throw IOException, need to check it in try catch block
the methods can throw only RuntimeException's

Reply:
pls fix in all methods of this class